packages/frontend/components/forms/sign-up.js

```diff
@@ -32,8 +34,10 @@ function Signup() {
       }}
       onSubmit={(values, { setSubmitting }) => {
         setTimeout(() => {
+          // todo: delete console.log
           // eslint-disable-next-line no-console
-          console.log(JSON.stringify(values, null, 2));
+          console.log(values);
```

Comment:
why do we need to console log this?

Reply:
actually, we don't. its temporary. there is no place to check form values is valid or not at the moment. i can't be sure about values since we are using customized formik form fields and onChange handlers.

i thought that deleting it after back end ready is good idea.

Reply:
why not use https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en ?